keras_hub/src/models/whisper/whisper_audio_converter.py

```diff
@@ -214,21 +219,21 @@ def call(self, audio):
 
         rank_1_input = audio.shape.rank == 1
         if rank_1_input:
-            audio = tf.expand_dims(audio, 0)
+            audio = ops.expand_dims(audio, 0)
 
         # Convert the tensor to a Ragged Tensor.
         if isinstance(audio, tf.Tensor):
```

Comment:
We could probably switch to a `@preprocessing_function` annotation and remove this? Something to try at least.